backend/app/schemas/content.py

```
@@ -49,4 +50,25 @@ class ContentItemDetail(ContentItemPublic):
 # Schemas for ContentAsset (can be added later if needed for API endpoints)
 # For this task, only ContentItem schemas are explicitly required for the endpoints.
 
-print("Schemas for ContentItem created in backend/app/schemas/content.py")
+
+# Schemas for ContentShare
+class ContentShareBase(SQLModel):
+    expires_at: Optional[datetime] = Field(default=None)
+    max_access_count: Optional[int] = Field(default=None)
+    password: Optional[str] = Field(default=None, sa_column_kwargs={"exclude": True}) # Write-only
```

Ruff (0.11.9)

Use `X | Y` for type annotations

Convert to `X | Y`

(UP007)